src/test/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineArtifactStagingTest.java

```diff
@@ -89,7 +94,7 @@ public void testCopyDockerContext_runtimeJavaNoWarning() throws AppEngineExcepti
   public void testCopyDockerContext_noDocker() throws AppEngineException, IOException {
     when(config.getDockerDirectory()).thenReturn(null);
 
-    CloudSdkAppEngineFlexibleStaging.copyDockerContext(config, copyService, "java");
+    CloudSdkAppEngineArtifactStaging.copyDockerContext(config, copyService, "java");
```

Comment:
What ArtifactStaging do we have that is not CloudSDK based?

Reply:
This is a legacy thing I guess. There's the Interfaces which can be removed
```
CloudSdkAppEngineArtifactStaging implements AppEngineArtifactStaging
```
but I'm not covering that in this PR.